Tests/Functional/RequestBodyParamConverterTest.php

```diff
@@ -28,4 +28,13 @@ public function testRequestBodyIsDeserialized()
         $this->assertSame(200, $client->getResponse()->getStatusCode());
         $this->assertSame('Post 1', $client->getResponse()->getContent());
     }
+
+    /**
+     * @see https://github.com/FriendsOfSymfony/FOSRestBundle/issues/1237
+     */
+    public function testTwigErrorPage()
+    {
+        $client = $this->createClient(['test_case' => 'RequestBodyParamConverter']);
+        $client->request('GET', '/_error/404');
```

Comment:
I think so. Checking for the proper status code (404 instead of 500) should be enough though.